scripts/install_third_party.py

```diff
@@ -164,6 +165,49 @@ def return_json(source_url):
     response = get_file_contents(source_url)
     return json.loads(response)
 
+def test_manifest_zip(source_contents):
+    keys = source_contents.keys()
+    mandatory_keys = ['version', 'zipUrl']
+    optional_keys_pairs=[['rootDir', 'rootDirPrefix'], ['targetDir', 'targetDirPrefix']]
+    for key in mandatory_keys:
+        if key not in keys:
+            print '------------------------------------------'
+            print 'There is syntax error in this dependency'
+            print source_contents
+            print 'This key is missing or mispelled: "%s".' %key
+            print 'Exiting'
+            sys.exit()
+    for optional_keys in optional_keys_pairs:
+        if not any([True for key in optional_keys if key in keys]):
+            sys.exit()
+
+
+def test_manifest_tar(source_contents):
+    keys = source_contents.keys()
+    mandatory_keys = ['version', 'tarUrl', 'tarRootDirPrefix', 'targetDirPrefix']
+    for key in mandatory_keys:
+        if key not in keys:
+            print '------------------------------------------'
+            print 'There is syntax error in this dependency'
+            print source_contents
+            print 'This key is missing or mispelled: "%s".' %key
```

Comment:
nit, here and elsewhere: add single space after '%'.

Also, 'misspelled' is misspelled :-)